CHANGELOG.md

```diff
@@ -15,6 +15,40 @@ From version 8.0.0 onwards, [Semantic Versioning](http://semver.org/) is used.
 _Nothing yet._
 
 
+## [8.1.0] - 2017-12-17
+
+See all related issues and PRs in the [8.1.0 milestone].
+
+### Added
+- :star2: New `MagicAutoloadDeprecation` sniff to detect deprecated __autoload functions as deprecated in PHP 7.2. [#540](https://github.com/wimg/PHPCompatibility/pull/540)
+- :star2: New `DynamicAccessToStatic` sniff to detect dynamic access to static methods and properties, as well as class constants, prior to PHP 5.3. [#535](https://github.com/wimg/PHPCompatibility/pull/535)
+- :star2: New `NewConstants` and `RemovedConstants` sniffs to detect usage of new/removed PHP constants. [#526](https://github.com/wimg/PHPCompatibility/pull/525)
+- :star2: New `OptionalRequiredFunctionParameter` sniff to check for missing function call parameters which were required and only became optional in a later PHP version. [#524](https://github.com/wimg/PHPCompatibility/pull/524)
+- :star: `RemovedGlobalVariables` : detection of the use of $php_errormsg with track_errors. [#528](https://github.com/wimg/PHPCompatibility/pull/528)
+- :star: `NewFunctions` : detection of all new PHP 7.2 functions added. [#522](https://github.com/wimg/PHPCompatibility/pull/522)
+- :star: Build/PHPCS : added array indentation check. [#538](https://github.com/wimg/PHPCompatibility/pull/538)
```

Comment:
I don't mind detailing the separate ones. It is a minor release after all, not a patch release ;-)
I'm leaving the ones that fix things that were broken (even if it's in the build process) under Fixed, since they are fixes to the PHPCompatibility project as such.